migrate/src/main/scala/migrate/utils/Timer.scala

```diff
@@ -0,0 +1,16 @@
+package migrate.utils
+
+import scala.util.control.NonFatal
+import scala.util.{Success, Failure, Try}
+import scribe._
+import scala.language.experimental.macros
+import scala.reflect.macros.blackbox
+import scala.annotation.compileTimeOnly
+
+private[migrate] object Timer {
```

Comment:
I would use FIniteDuration 
```
  def timedMs[A](task: => A): (A, FiniteDuration) ={
    val start =  Instant.now()
    val result = task

    (result, toFiniteDuration(start, Instant.now()))
  }

  def toFiniteDuration(start: Instant, end: Instant): FiniteDuration =
    FiniteDuration(end.toEpochMilli - start.toEpochMilli, MILLISECONDS).toCoarsest
```
the toString will be by default to milliseconds (no need to add `ms` in logging messages)

Reply:
That's a good idea!